### ISSUE TITLE
Nom-O-Matic 25000-I stops working when vessel loses focus

### ISSUE THREAD

**Issue by Neumanf15** [BobPalmer/USI-LS#204]
For an unknown reason, the Nom-O-Matic 25000-I stops producing Supplies when it is out of focus.  Reproduction:  
1. Build a vessel with whatever hab parts are required and a Nom-O-Matic 25000-I with supplies and fertilizer.  
2. Launch the vessel with a crew that consumes less than 4.75 supplies per hour.  
3. Ensure all LS functions are active and Nom-O-Matic 25000-I is running agroponics.  
4. Return to space center and warp forward several days.  
5. Return to the test vessel and note that the supplies are not full.  
6. Time warp with vessel in focus and the supplies fill back up.

**Comment by BobPalmer** [comment]
Please attach a save with an example vessel and a quicksave.

**Comment by Neumanf15** [comment]
Sorry, took a few minutes to make this with minimum mods.  Only had your mods, EL, KAS, KIS and hyperedit installed when I made this for you.

Quicksave was made after 24 days in times warp with the "Nom-O-Matic Test" ship in focus.  From the quicksave, all I did was change ships, extend the solar panels on "Relay" and time warp.  Persistent is after 15 days with the other ship ("Relay") in focus.

[NOM-O-MATIC.zip](https://github.com/BobPalmer/USI-LS/files/706995/NOM-O-MATIC.zip)

**Comment by Neumanf15** [comment]
Further info:  I think the problem is NOT actually the Nom-O-Matic.  In fact, I think the problem is with recyclers!
I just updated to your latest releases and ran this test:
From the quicksave I sent you, I experimented with different recycler configurations.  I added in Kerbal alarm clock to control the test to be exactly 10days and 10 minutes.  I ran each configuration in focus and again out of focus.  I've attached the results.  
The first three configurations have enough recyclers for ALL of the Kerbals.  All have the expected result when in focus:  Supplies full, fertilizer goes down by the amount that mulch goes up - consistent with the supplies needed by the kerbals during the 10 days.  When out of focus, all three act like there are no recyclers - they basically match the no recycling lines - except they are short 25 mulch of expectations which may be tied to when you do the catch-up math.
The two "only" configurations had very weird results!  Both of these configurations have less recyclers than Kerbals.  While in focus, fertilizer went down by the expected amount - mulch went up by the same and supplies remained full.  While out of focus, however...  They used LESS than expected fertilizer (mulch still matched fertilizer) and they remained at the expected 300 supplies.
The last two lines are controls with no recyclers active.  Results are as expected except for the missing 25 mulch I previously mentioned.
![results](https://cloud.githubusercontent.com/assets/25129262/22003806/6c638436-dc0b-11e6-92a9-bf831d3e512f.JPG)
